docs/BENCHMARKS.md

```diff
@@ -145,9 +145,9 @@ asv profile benchmark_qt_viewer.QtViewerSuite.time_create_viewer -g snakeviz --p
 or
 
 ```
-asv profile "benchmark_image_layer.Image2DSuite.time_create_layer(.*)" -g snakeviz --python=same
+asv profile "benchmark_image_layer.Image2DSuite.time_create_layer\(512\)" -g snakeviz --python=same
 ```
-to profile a parametrized benchmark.
+to profile a particular parametrized benchmark.
```

Comment:
At the risk of being too verbose, I think it might be nice to add a bit of text explaining what is being done here. For example:

```suggestion
where `benchmark_image_layer` is the file name, `Image2DSuite` is the test suite class name, and `time_to_create_layer()` is the test method.
```

and change the "or" above to something like. "To profile a particular parameterized benchmark, run

Reply:
nice idea, I've reworded it a little, but the new language should capture your desired change